docs/en/operations/table_engines/log.md

```diff
@@ -1,5 +1,7 @@
 # Log
 
+Engine belongs to the [Family of log engines](log_family.md).
```

Comment:
This could say that it inherits common properties of log engine family, so see the main article to learn about them.